src/shader/shaderLightSampling.js

```diff
@@ -27,20 +67,50 @@ LightSampleRec lightsClosestHit( sampler2D lights, uint lightCount, vec3 rayOrig
 			continue;
 		}
 
-		u *= 1.0 / dot(u, u);
-		v *= 1.0 / dot(v, v);
+		u *= 1.0 / dot( u, u );
+		v *= 1.0 / dot( v, v );
```

Comment:
Are these variables and the normal backface check above still required for spot lights? Or should they be moved into the "Area Light" condition?